mama/c_cpp/src/examples/cpp/mamapublishercpp.cpp

```diff
@@ -51,7 +51,6 @@ using std::cout;
 using std::endl;
 
 static int           gCount         = 0;
-static const char *  gSource        = "";
```

Comment:
Because the normal purpose of the example app is to demonstrate basic subscriptions in conjunction with mamasubscribercpp, and basic subscriptions do not use a source.